models/a-2084-01/model.config

```diff
@@ -1,9 +1,9 @@
 
 <?xml version="1.0"?>
 <model>
-  <name>HEBI A-2084-01</name>
+  <name>HEBI a-2084-01</name>
```

Comment:
I was originally uncertain about the file name change (`A` -> `a`), but I think I'm comfortable with it overall.  However, can we keep the name tags with the capitalization matching the website docs (A-2084-01, etc), here and other places?

Reply:
I have done some extra testing to see why I did it this way. Some of the moveit config launch/yaml files seem to base their naming scheme off the model name, rather than the model's filename. I can make this change so that the name stays capitalized, but that means that some of the files will be uppercase, and some lowercase. This isn't a _huge_ deal since all the configs are generated automatically by the Moveit! wizard but I still don't like the mix of cases between urdf, srdf, sdf, and the launchfile includes. Maybe we can chat about this more later when we're discussing other naming things.